src/api/channels.tsx

```diff
@@ -150,9 +155,10 @@ export const constructColumnDefs = (
         return <React.Fragment>{wordWrap.join('')}</React.Fragment>;
       },
       meta: { channelInfo: channel },
-      cell: isChannelMetadataScalar(channel)
-        ? ({ getValue }) => {
-            const value = getValue();
+      cell: ({ row, getValue }) => {
+        const value = getValue<string>();
```

Comment:
Technically this type is wrong in the scalar case which can be a number. Not sure this really matters though...